server/config/incident.js

```diff
@@ -23,6 +25,15 @@ const paths = {
     `/request-removal/${statementId}${signature ? `?signature=${signature}` : ''}`,
   alreadyRemoved: () => '/already-removed',
   removalRequested: () => '/removal-requested',
+  viewRemovalRequest: (reportId, statementId) =>
+    `/coordinator/report/${reportId}/statement/${statementId}/view-removal-request`,
+  staffMemberNotRemoved: (reportId, statementId) =>
+    `/coordinator/report/${reportId}/statement/${statementId}/staff-member-not-removed`,
+  confirmStatementDelete: (reportId, statementId, removalRequest) =>
+    `/coordinator/report/${reportId}/statement/${statementId}/confirm-delete${
+      removalRequest ? `?removalRequest=${removalRequest}` : ''
+    }`,
+  confirmReportDelete: reportId => `/coordinator/report/${reportId}/confirm-delete`,
```

Comment:
I've been sort of misusing this - this is supposed to be a "create report" specific file.. we could probably do with either extracting the routes out into somewhere else or breaking down based on application section ("createReport", "manageReports" etc...)

This is obviously completely fine as it is though.. that would be done separate PR